cross_build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Is the inclusion of this file necessary?

Reply:
This file is unnecessary, I remove it.